.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -98,6 +98,20 @@ The integration reached or maintains the following [Integration Quality Scale][q
 - [ ] 🥇 Gold
 - [ ] 🏆 Platinum
 
+<!--
+  This project is very active and we have a high turnover of pull requests.
+  Unfortunately the number of incoming pull requests is higher than what our
+  reviewers can review and merge so there is a long backlog of pull requests
+  waiting for review. You can help here! By reviewing another pull request
+  you will help raise the code quality of that pull request and the final
+  review will be faster. This way the general pace of pull request review
+  will go up and your wait time will go down. Thanks for helping out!
```

Comment:
I love this!

I've thought about this before myself, since I've been confronted with a repo that did similar (only forced/required).

Example: <https://github.com/sindresorhus/awesome/pull/1838>

![image](https://user-images.githubusercontent.com/195327/90234657-4ff50980-de20-11ea-8496-1da0bef47493.png)

I think we can learn a thing or 2 from this one.

- It adds a link on the "open pull requests" term (should sort the oldest first?)
- It tells the contributor actively to pick up unreviewed ones
- "Just commenting “looks good” or simply marking the pull request as approved does not count! You have to actually point out mistakes or improvement suggestions." might be a good one?

Reply:
I'd like to try this without adding compulsion in the language. We're not enforcing the check yet, although we could add some automation to prioritize PRs that have checked this.

Points 1-2 are good!

Reply:
Yeah, I agree, it should not be forced or feel like it is forced.

Reply:
I wonder, do people even realize they can leave reviews, even if they are not a member of the project?

Reply:
No, many people don't know that they can leave reviews. That's the first thing we can try to improve with this checkbox.

Reply:
```suggestion
  This project is very active and we have a high turnover of pull requests.
  Unfortunately the number of incoming pull requests is higher than what our
  reviewers can review and merge so there is a long backlog of pull requests
  waiting for review. You can help here! By reviewing another pull request
  you will help raise the code quality of that pull request and the final
  review will be faster. This way the general pace of pull request review
  will go up and your wait time will go down. When picking a pull request to
  review, try to choose one that hasn't yet been reviewed.
  Thanks for helping out!
```

Reply:
Same text as your suggestion, added a comma or 2.

Maybe a of enters help to make it more readable?

```suggestion
  This project is very active and we have a high turnover of pull requests.

  Unfortunately, the number of incoming pull requests is higher than what our
  reviewers can review and merge so there is a long backlog of pull requests
  waiting for review. You can help here!
  
  By reviewing another pull request, you will help raise the code quality of
  that pull request and the final review will be faster. This way the general
  pace of pull request reviews will go up and your wait time will go down.
  
  When picking a pull request to review, try to choose one that hasn't yet
  been reviewed.

  Thanks for helping out!
```

Reply:
Thanks!